test/test_utils.py

```diff
@@ -611,6 +611,28 @@ def test_timer(self):
         median = timer.blocked_autorange(min_run_time=0.1).median
         self.assertIsInstance(median, float)
 
+    def test_adaptive_timer(self):
+        # Validate both on different sizes validate against blocked_autorange
+        # This looks for relative differences btetween orders of magnitude to
+        # provide a stable/portable test which is somewhat informative.
+        timer = benchmark_utils.Timer(
+            stmt="torch.sum(torch.ones((10,10)))",
+        )
+        small = timer.adaptive_autorange(min_run_time=0.1).median
+        timer = benchmark_utils.Timer(
+            stmt="torch.sum(torch.ones((100,100)))",
```

Comment:
Could you make this `(500, 100)`? Despite being 100x apart, this one only takes 1.5x as much time due to the `(10, 10)` case being overhead bound. And that should still leave plenty of space between medium and large.

Reply:
done.